python/addPFCands_cff.py

```diff
@@ -71,6 +73,8 @@ def addPFCands(process, runOnMC=False, allPF = False, onlyAK4=False, onlyAK8=Fal
                                                         idx_name = cms.string("pFCandsIdx"),
                                                         nameSV = cms.string("JetSVs"),
                                                         idx_nameSV = cms.string("sVIdx"),
+																												#nameMu = cms.string("JetMuons"),
+                                                        #idx_nameMu = cms.string("MuIdx"),
```

Comment:
[Style] Places with commented out code which could be removed for clarity
The default name JetMuons and Idx when adding the Muon table to AK4 jets could also be removed (currently commented out).